Tasks/AzureRmWebAppDeployment/Utility.ps1

```diff
@@ -170,6 +195,34 @@ function Run-Command
 
 }
 
+function Get-CommandOutput
```

Comment:
Task lib has this method -> [invoke-vststool](https://github.com/Microsoft/vsts-task-lib/blob/master/powershell/Docs/Commands.md#invoke-vststool) which sets one environment variable with status code but doesn't gives back result of execution.